src/lapi.cc

```diff
@@ -126,6 +128,59 @@ namespace blt {
             return 1;
         }
 
+        int
+        movedir(lua_state* state)
+        {
+            size_t fromLen, toLen;
+            const char* from = lua_tolstring(state, 1, &fromLen);
+            const char* to   = lua_tolstring(state, 2, &toLen);
+
+            if(lua_gettop(state) != 2)
+                luaL_error(state, "file.MoveDirectory(from, to) takes two arguments, not %d", lua_gettop(state));
+
+            if(strlen(from) != fromLen)
+                luaL_error(state, "file.MoveDirectory(from, to): argument 'from' cannot contain null characters!");
+
+            if(strlen(to) != toLen)
+                luaL_error(state, "file.MoveDirectory(from, to): argument 'to' cannot contain null characters!");
+
+            bool result = rename(from, to) == 0;
```

Comment:
would prefer `(rename() == 0)` fwiw.

lgtm tho

Reply:
I'll do that quickly